src/features/error/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Space, Typography } from 'antd';
+import logo from 'assets/logo.png';
+import { ErrorWrapper, ErrorHeader, LargeWidthImage } from './styled';
+
+const Error = () => {
+  const { Title, Link } = Typography;
```

Comment:
I'm unsure if this should be inside the `Error` function or outside. Either works, but I'm going to ping @nhivn for input here.

Reply:
Hello. Any update on this? :)

Reply:
Sorry! Yes we should bring it out. Once thats done I can approve :)